### PR TITLE
Add issue template for website removal requests

### PR DESCRIPTION
**Pull request by 27136773**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/5-website-removal-request.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+name: Website Removal Request
```
Comment (Sheepux) [inline .github/ISSUE_TEMPLATE/5-website-removal-request.md]: Add emoji icon in the name 
name: ⛔ Website Removal Request

**Review by Sheepux — APPROVED** [review]
/LGTM if you add the emoji :)

File: .github/ISSUE_TEMPLATE/5-website-removal-request.md (hunk 1 above)
Comment (27136773) [reply .github/ISSUE_TEMPLATE/5-website-removal-request.md]: Added

**Comment by Mynamevn** [comment]
Hi, all vietnamese websites are dead. please remove them. Thanks.

**Pull request merged**